CMakeLists.txt

```diff
@@ -64,10 +66,11 @@ endif()
 
 file(GLOB HEADERS "include/chainbase/*.hpp")
 add_library( chainbase src/chainbase.cpp src/pinnable_mapped_file.cpp ${HEADERS} )
-target_link_libraries( chainbase ${PLATFORM_LIBRARIES} Boost::boost)
+target_link_libraries( chainbase ${PLATFORM_LIBRARIES}
+                        Boost::headers Boost::interprocess Boost::chrono Boost::multi_index Boost::lexical_cast Boost::asio )
```

Comment:
most of these targets don't exist in the non-cmake boost install.

Reply:
But they do seem needed in the boost cmake use case. Do you know of a way around it?